site/src/pages/TemplateSettingsPage/TemplateSettingsForm.tsx

```diff
@@ -188,9 +187,7 @@ export const TemplateSettingsForm: FC<TemplateSettingsForm> = ({
         there are no validation errors for that field, display helper text.
         We do not use the MUI helper-text prop because it overrides the validation error */}
         {form.values.default_ttl_ms && !form.errors.default_ttl_ms && (
-          <Typography variant="subtitle2">
-            {Language.ttlHelperText(form.values.default_ttl_ms)}
-          </Typography>
+          <span>{Language.ttlHelperText(form.values.default_ttl_ms)}</span>
```

Comment:
Changed this because I thought it was too big. Now it matches the other helper text in the form.